recipes/perl-ostype/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "perl-perl-ostype" %}
+{% set version = "1.010" %}
+{% set sha256 = "e7ed4994b5d547cb23aadb84dc6044c5eb085d5a67a6c5624f42542edd3403b2" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/D/DA/DAGOLDEN/Perl-OSType-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
```

Comment:
host

Reply:
oh. I just created this recipe with ``conda skeleton cpan Perl::OSType``. This means the skeleton script needs an update.

Reply:
I guess so ... or do you have an old version of conda-build?